app/src/main/java/com/conupods/Settings/SettingsActivity.java

```diff
@@ -0,0 +1,98 @@
+package com.conupods.Settings;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.SharedPreferences.Editor;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.CheckBox;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.conupods.R;
+
+public class SettingsActivity extends AppCompatActivity {
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.settings_page);
+
```

Comment:
If you are leaving extra lines here for readability, you should write a short comment in place of the blank line (see also like 25, and 32)